### PR TITLE
feature: listen to error emitted(request,send,pub)

### PR DESCRIPTION
**Pull request by yamii**

### CODE REVIEW

File: test/send-receive-test.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ describe( 'send and receive', function () {
 
 			before( function ( done ) {
 
+				RabbusStub.Sender.prototype.once    = function ( type, callback ) {};
```
Comment (openam) [inline test/send-receive-test.js]: Would you fix the alignment?

**Comment by openam** [comment]
Do we need to update the package version? Should this just go to 3.0.1?

File: test/send-receive-test.js (hunk 1 above)
Comment (yamii) [inline test/send-receive-test.js]: :+1:

**Comment by yamii** [comment]
@openam i'm OK in tagging this to 301 patch

**Comment by openam** [comment]
What do you mean by `tagging this to 301 patch`? I was talking about the version in the `package.json` file.

**Comment by MrAnderson0309** [comment]
Before we can merge this in can we get it to log out in a log file what is not being responded to  and any info that would help is find out why

**Comment by yamii** [comment]
@openam what i've understand from your first comment is to put the changes into v3.0.1. Do I understand it right?

File: lib/pub-sub.js [hunk 2]
```diff
@@ -4,6 +4,16 @@ var util   = require( 'util' );
 var Rabbus = require( 'rabbus' );
 var mixins = require( './mixins' );
 
+function onErrorEmit ( rabbit, callback ) {
+	rabbit.once( 'error', function ( reqresError ) {
```
Comment (openam) [inline lib/pub-sub.js]: @yamii @fakewaffle Does this mean that it only listens for the first error?

**Pull request merged**